cli/lsp/language_server.rs

```diff
@@ -1632,23 +1632,30 @@ impl Inner {
     &self,
     params: DocumentFormattingParams,
   ) -> LspResult<Option<Vec<TextEdit>>> {
-    let specifier = self
+    let mut specifier = self
       .url_map
       .normalize_url(&params.text_document.uri, LspUrlKind::File);
+    // skip formatting any files ignored by the config file
+    if !self.fmt_options.files.matches_specifier(&specifier) {
```

Comment:
If someone ignores something in the vendor folder, will this also skip those files? I think probably not. Maybe it should?

Reply:
Currently `vendor` isn't applicable to config exclusion checks because it's excluded by default. We should remove https://github.com/denoland/deno_config/blob/0.4.0/src/lib.rs#L688-L690 for a start. It shouldn't be handled there.